pkg/okta/okta.go

```diff
@@ -29,23 +33,39 @@ func (f oktaMiddlewareFactory) jwt(logger *zap.Logger, authHeader string) (*jwtv
 	return f.verifier.VerifyAccessToken(bearerToken)
 }
 
-func (f oktaMiddlewareFactory) newUser(logger *zap.Logger, jwt *jwtverifier.Jwt) (models.User, error) {
+func (f oktaMiddlewareFactory) newPrincipal(jwt *jwtverifier.Jwt) (*authn.EUAPrincipal, error) {
 	euaID := jwt.Claims["sub"].(string)
 	if euaID == "" {
-		return models.User{}, errors.New("unable to retrieve EUA ID from JWT")
+		return nil, errors.New("unable to retrieve EUA ID from JWT")
+	}
+
+	// the current assumption is that anyone with an appropriate
+	// JWT provided by Okta for EASi is allowed to use EASi
+	// as a viewer/submitter
+	jcEASi := true
+
+	// need to check the claims for empowerment as a reviewer
+	jcGRT := false
+
+	// TODO: this is a placeholder implementation that will
```

Comment:
note the placeholder code until we see where the job codes will actually live on the JWT.